src/lib/spinel/spinel.h

```diff
@@ -3561,6 +3561,22 @@ enum
      */
     SPINEL_PROP_STREAM_LOG = SPINEL_PROP_STREAM__BEGIN + 4,
 
+    /// CLI Stream
+    /** Format: `UD` - Set (Host to NCP) and Unsolicited Notifications (NCP to Host)
+     *
+     * This streaming property provides a bidirectional channel for interacting with the NCP's
+     * command-line interface (CLI).
+     *
+     *   `U`: utf8 string from Host to NCP
+     *   `D`: output data from NCP to Host
```

Comment:
I think the format is always just a single `U` (one string) and not `UD`?

Reply:
It appears that within `VALUE_SET`, the property is handled as `U` using `mDecoder.ReadUtf8()`. But for `VALUE_IS` notification, it's treated as `D` using `mEncoder.WriteDataWithLen`.

Generally, a Spinel property has a common value format, irrespective of whether we get or set its value. 

Suggest changing this to use the same format for `VALUE_SET` and `VALUE_IS`. Make sense to use `Utf8` format in both cases.